README.md

```diff
@@ -445,26 +445,40 @@ Hazelcast Node.js client supports the following data structures and features:
 * Queue
 * Set
 * List
-* Multi Map
-* Replicated Map
-* Ringbuffer
+* MultiMap
+* ReplicatedMap
+* RingBuffer
 * Reliable Topic
-* Near Cache support for Map
 * Lock
 * Semaphore
 * Atomic Long
+* CRDT PN Counter
 * Flake Id Generator
-* Fast Aggregations
-* CRDT Counter
 * Event Listeners
-* Entry Processors
-* Predicates
+* Entry Processor
+* Query (Predicates)
+* Paging Predicate
+* Built-in Predicates
+* Listener with Predicate
+* Fast Aggregations
+* Near Cache Support
+* Eventual Consistency Control
+* Declarative Config (JSON)
+* Programmatic Config
+* Client Configuration Import
+* Fail Fast on Invalid Configuration
+* SSL Support (requires Enterprise server)
+* Mutual Authentication (requires Enterprise server)
+* Authorization
 * Smart Client
 * Unisocket Client
-* Hazelcast Native Serialization
 * Lifecycle Service
-* SSL connection support (requires enterprise server)
+* Heartbeat Service
```

Comment:
Is it necessary for user ?

Reply:
I dont think so. I just added it because it was a thing in client feature comparison.